lib/rubycritic/commands/default.rb

```diff
@@ -20,6 +26,15 @@ def critique
         RevisionComparator.new(@paths).set_statuses(analysed_modules)
       end
 
+      def open(report_path)
```

Comment:
This method will only be called when we use a html_formatter, right?
This means there will **always** be a report_path, won't it?
Cause then we can drop the argument check and delegate this call directly to `browser`.